app/assets/stylesheets/application.css

```diff
@@ -1,5 +1,5 @@
 /*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
+ * This is a manifest file that'll be compiled into et_dropzone_uploader.css, which will include all the files
```

Comment:
Same as the js - why et_dropzone_uploader ?  this is the app not the gem isnt it ?

Reply:
Good point - fixed in b055fec thank you!